lib/server/plugins/api/index.js

```diff
@@ -118,10 +118,18 @@ exports.register = function (plugin, options, next) {
     );
   };
 
+  internals.maybeForceGzip = function (request, reply) {
+    if (request.query.gzip === 'force') {
+      request.info.acceptEncoding = 'gzip';
+    }
+    reply();
+  };
+
   plugin.route([
     {
       method: 'OPTIONS',
       path: '/_api/{p*}',
+      config: { pre: [{ method: internals.maybeForceGzip }] },
```

Comment:
Fixed thanks to @wraithgar and @nlf for [solving this to me here](https://github.com/hoodiehq/hoodie-server/commit/4169d0f6af6506e7f151ba4e409cd7c383f412c6#diff-e09bca3f48fed5aa8460eb81b1612dc8R125)! 💕👊✨